ui/app/templates/partials/tools/unwrap.hbs

```diff
@@ -8,19 +8,43 @@
 
 {{#if unwrap_data}}
   <div class="box is-sideless is-fullwidth is-marginless">
-    <div class="field">
-      <label class="is-label">
-        Unwrapped data
-      </label>
-      <div class="control">
-        {{json-editor
-          value=(stringify unwrap_data)
-          options=(hash
-            readOnly=true
-          )
-        }}
+    <nav class="tabs">
+      <ul>
+        <li role="tab" aria-selected={{if (eq unwrapActiveTab "data") "true" "false"}} class="{{if (eq unwrapActiveTab "data") "is-active"}}" {{action (mut unwrapActiveTab) "data" on="click"}}>
```

Comment:
I think there should be `<button>`s with the actions inside these `li`s - `li` isn't an interactive element so it feels kinda weird to add click handlers to it.